src/utils/strings.ts

```diff
@@ -32,3 +32,12 @@ export const copyToClipboard = async (text: string): Promise<void> => {
 
 export const formatCost = (cost: Payment, token?: string): string =>
```

Comment:
maybe an idea to include a small comment explaining what it does, because I am not sure what it does.